letting_context.md

```diff
@@ -175,6 +181,155 @@ A user might change the reference of a unit in GARAIO REM which affects tenancy
 }
 ```
 
+### Letting.TenancyAgreement.DraftCreated
+
+The tenancy agreement has become validated. 
+
+| Field                 | Type     | Content / Remarks                                                                                           |
+|-----------------------| -------- |-------------------------------------------------------------------------------------------------------------|
+| eventType             | `string` | Letting.TenancyAgreement.DraftCreated                                                                       |
+| data                  | `hash`   |                                                                                                             |
+| &nbsp;&nbsp;reference | `string` | unique tenancy agreement identifier, eg '1234.01.0001.01'                                                   |
+| &nbsp;&nbsp;startDate | `string` | The starting date of the tenancy agreement.                                                                 |
+| &nbsp;&nbsp;state     | `string` | The current state of the tenancy agreement. It can be in validated, activated, deactivated, cancelled state |
+
+#### Example
+
+```json
+{
+  "eventType":"Letting.TenancyAgreement.DraftCreated",
+  "data": {
+    "reference":"1234.01.0001.01",
+    "startDate":"2025-08-03",
+    "state":"validated"
+  }
+}
+```
+
+### Letting.TenancyAgreement.DraftUpdated
+
+The validated tenancy agreement has some changes. If the starting date of the tenancy has been updated, we
+include a new key `newStartDate` in the message body and send the old starting date in the `startDate` key. 
+
+| Field                    | Type     | Content / Remarks                                                                                                                     |
+|--------------------------| -------- |---------------------------------------------------------------------------------------------------------------------------------------|
+| eventType                | `string` | Letting.TenancyAgreement.DraftUpdated                                                                                                 |
+| data                     | `hash`   |                                                                                                                                       |
+| &nbsp;&nbsp;reference    | `string` | unique tenancy agreement identifier, eg '1234.01.0001.01'                                                                             |
+| &nbsp;&nbsp;startDate    | `string` | The starting date of the tenancy agreement. If starting date of tenancy agreement changes, then here we expose the old starting date. |
```

Comment:
```suggestion
| &nbsp;&nbsp;startDate    | `string` | The starting date of the tenancy agreement. Always sent. If starting date of tenancy agreement changes, this will be the old starting date. |
```